_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Mesh Expansion
+overview: Instructions to add external machines and expanding Istio.
+
+order: 60
+
+layout: docs
+type: markdown
+---
+
+Instructions to configure Istio on a Kubernetes Cluster so it can be expanded with
+services running on cloud or on-prem VMs or external machines.
+
+## Prerequisites
+
+* You have [installed Istio](quick-start.html) on Kubernetes.
+
+* The machine must have IP connectivity to the endpoints in the mesh. This
+typically requires same VPC or a VPN connection, as well as a container network that
+provides direct (without NAT or firewall deny) routing to the endpoints. The machine
+is not required to have access to the cluster IP addresses assigned by K8S.
+
+* The control plane (Pilot, Mixer, CA) and Kubernetes DNS server must be accessible
+from the VMs. This is typically done using an [Internal Load
+Balancer](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer).
+Other options are possible, for example NodePort, or running Istio components on VMs, or
+custom network configurations - separate documents will cover advanced configs.
```

Comment:
"You can also use NodePort, run Istio components on VMs, or use custom network configurations. To use custom network configurations, see [doc]." <- is there a doc for this or is it yet to be written?

Reply:
changing now